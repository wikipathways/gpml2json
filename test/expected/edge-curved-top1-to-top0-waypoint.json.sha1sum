20fddf8010071d8e1d8a1cf10c5d26cd7ba36ffa ?-
