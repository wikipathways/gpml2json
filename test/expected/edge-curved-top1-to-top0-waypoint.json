{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c99cf","cc766","f2465"],"drawAs":"rect","gpmlElementName":"Pathway","height":783.1967827425314,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1045.87076968672,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/b3e850b694a88b254b4c681e58a2960285020a06b0cccf9ce58e533da601b6da"},"entityMap":{"cc766":{"id":"cc766","textContent":"GeneProduct","wpType":"GeneProduct","x":143.35885932305436,"y":130.43165027899875,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32945,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f2465":{"id":"f2465","textContent":"0001","wpType":"GeneProduct","x":16.576071349526885,"y":85.71948434602173,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32946,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c99cf":{"id":"c99cf","drawAs":"CurvedLine","borderWidth":1,"zIndex":32944,"points":[{"isAttachedTo":"f2465","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":56.576071349526885,"y":85.71948434602173,"orientation":[0,-1]},{"x":109.96746533629062,"y":56.07063628896775},{"isAttachedTo":"cc766","attachmentDisplay":{"position":[0.25,0],"offset":[0,0]},"x":163.35885932305436,"y":130.43165027899875,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["f2465","cc766"]}}}
